src/core/public/index.ts

```diff
@@ -360,3 +362,5 @@ export {
   UiSettingsState,
   NavType,
 };
+
+export { bootstrap } from './bootstrap';
```

Comment:
can we call this something like `__kbnBootstrap__` to make it obvious that plugins should not call this.